go/libkb/db.go

```diff
@@ -166,7 +166,8 @@ const (
 	DBUser                    = 0x00
 	DBSig                     = 0x0f
 	DBTeamChain               = 0x10
-	DBUserPlusAllKeys         = 0x19
+	DBUserPlusAllKeysV1       = 0x19
+	DBUserPlusKeysAIV2        = 0x1a
```

Comment:
Can't take `0x1a`, that already means `billing_plan`.  If it doesn't exist in `id.iced`, I'd use `0xe4`